packages/jest-validate/src/types.js

```diff
@@ -14,6 +14,7 @@ type Title = {|
 |};
 
 export type ValidationOptions = {
+  blacklist?: Array<string>,
```

Comment:
`recursiveBlacklist`? Or something like that, I think it should be tied to what the blacklist is for, so I don't have to look up the docs